test/11-api-binder.spec.ts

```diff
@@ -24,8 +25,7 @@ const initModels = async (obj: Dictionary<any>, filename: string) => {
 	// @ts-ignore
 	config.configJsonBackend = new ConfigJsonConfigBackend(schema, filename);
 	config.generateRequiredFields();
-	// @ts-expect-error using private properties
-	config.configJsonBackend.cache = await config.configJsonBackend.read();
+	(config.configJsonBackend as any).cache = await (config.configJsonBackend as any).read();
```

Comment:
I'd rather a ts-expect-error than `any`. If something changes I'd rather get informed via a compile time breakage than not at all.